CMakeLists.txt

```diff
@@ -406,6 +430,13 @@ if (IS_64BIT_BUILD EQUAL 1)
   endif (CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64)
   add_definitions(-D_TARGET_AMD64_=1)
   add_definitions(-DDBG_TARGET_AMD64)
+elseif (CLR_CMAKE_PLATFORM_ARCH_ARM)
+  if (CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM)
+    add_definitions(-DDBG_TARGET_ARM_UNIX)
+  endif (CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM)
+  add_definitions(-D_TARGET_ARM_=1)
+  add_definitions(-DDBG_TARGET_ARM)
+  add_definitions(-DSTUBLINKER_GENERATES_UNWIND_INFO)
```

Comment:
It seemed like the ARM code quite heavily depended upon having the define, I could coerce it to build by just including the additional members in StubLinkCPU, though I don't really know what is the more correct approach here.